src/app/clients/form/formSections/NumberAdultsCard.tsx

```diff
@@ -1,59 +1,44 @@
 import React from "react";
 import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
 import {
-    Errors,
     errorExists,
     errorText,
     numberRegex,
-    NumberAdultsByGender,
+    Person,
+    Gender,
+    onChangeText,
 } from "@/components/Form/formFunctions";
 import GenericFormCard from "@/components/Form/GenericFormCard";
 import { SelectChangeEventHandler } from "@/components/DataInput/inputHandlerFactories";
-import { GappedDiv } from "@/components/Form/formStyling";
-import { ClientCardProps, ClientErrorSetter, ClientSetter } from "../ClientForm";
+import { FormText, StyledCard } from "@/components/Form/formStyling";
+import { ClientCardProps, ClientSetter } from "../ClientForm";
+import DropdownListInput from "@/components/DataInput/DropdownListInput";
+import { getAdultBirthYears } from "@/app/clients/form/birthYearDropdown";
 
-const getNumberAdultsOfGenderDefault = (numberAdultsOfGender: number): string | undefined => {
-    return numberAdultsOfGender === 0 ? undefined : numberAdultsOfGender.toString();
-};
+const minimumNumberOfAdults = 1;
+const maximumNumberOfAdults = 20;
```

Comment:
I think typescript naming convention for constant values is SCREAMING_SNAKE_CASE

Reply:
Never heard of that term! 🤣